src/main/java/org/mifos/connector/gsmastub/api/StatemententriesApiController.java

```diff
@@ -20,6 +20,7 @@
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2022-09-27T11:46:46.417Z[GMT]")
 @RestController
+@Hidden
```

Comment:
@ankita10r : Please remove these hidden annotations when this is refactored into separate channel connector.